### ISSUE TITLE
Project Server Output

### ISSUE THREAD

**Issue by Kaffiend** [NathanWalker/angular-seed-advanced#391]
```
[* ] feature request
```

**Current behavior**
Currently the server is still tightly coupled with the build tasks.

**Desired behavior**
Allow for new extended project server to be output to dist directory and spooled up with build tasks.

**What is the motivation / use case for changing the behavior?**
Ability to build decoupled API within the same project. This would be very nice for the electron aspect of the seed. a client server application, or nativescript for that matter, one server multiple clients.

**Comment by NathanWalker** [comment]
Agreed. @Kaffiend I won't have capacity for this request however the community may can help.

**Comment by Thavarajan** [comment]
hi @Kaffiend 
 any positive sign

**Comment by Kaffiend** [comment]
Unfortunately no, not yet. Been swamped here lately.
